src/Data/Language.hs

```diff
@@ -147,12 +148,18 @@ newtype PerLanguageModes = PerLanguageModes
   }
   deriving (Eq, Ord, Show)
 
+defaultLanguageModes :: PerLanguageModes
+defaultLanguageModes = PerLanguageModes
+  { pythonMode = ALaCarte
+  }
```

Comment:
I had initially extended the definition of `PerLanguageModes` to allow for java modes, but we don’t actually support à la carte Java terms any more anyway. This is still a convenient definition, however, and will only become more so as we migrate further languages over.